### PR TITLE
[p5.js HI] `access.md` and line 1 description translation to hindi

### PR DESCRIPTION
**Pull request by jaiakash**
Changes:
Translated `access.md` to Hindi and added Hindi translation of line 1 description on 4 pages.

#### PR Checklist

- [ ] `npm run lint` passes
- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Review by jaiakash — COMMENTED** [review]
Self review done

File: contributor_docs/hi/access.md [hunk 1]
```diff
@@ -1,40 +1,56 @@
-# पहुँच पर हमारा ध्यान
+<!-- लाइब्रेरी के योगदानकर्ताओं और उपयोगकर्ताओं के लिए पहुंच के प्रति हमारी प्रतिबद्धता का क्या मतलब है। -->
 
-[२०१९ के योगदानकर्ताओं के सम्मेलन](https://p5js.org/community/contributors-conference-2019.html) में, हमने पहुंच को बढ़ाने के लिए केवल p5.js में सुविधाओं को जोड़ने के लिए एक प्रतिबद्धता बनाई (जिसका अर्थ है समावेश / और / पहुंच क्षमता) । इसका अर्थ है विविधता के वैक्टर (जैसे लिंग, सामाजिक, आर्थिक, नस्ल, जातीयता, भाषा, विकलांगता, आदि) पर विचार करना जो पहुंच / भागीदारी को प्रभावित कर सकता है; और बाधाओं को स्वीकार, विघटित करने और रोकने के लिए कार्रवाई कर रहा है। हम p5.js. के साथ अधिक विशेषाधिकार प्राप्त समूहों के निरंतर आराम से अधिक ऐतिहासिक रूप से हाशिए वाले समूहों की आवश्यकताओं को प्राथमिकता देते हैं।
+# हमारा ध्यान सुगम्यता पर है
```
Comment (Divyansh013) [inline contributor_docs/hi/access.md:3]: I guess the removed line "पहुँच पर हमारा ध्यान" sounds better

File: contributor_docs/hi/access.md [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
+पहुंच बढ़ाना p5.js समुदाय में लोगों की संख्या का विस्तार करने पर केंद्रित नहीं है। यह उन लोगों के लिए p5.js को उपलब्ध कराने और उन तक पहुंच योग्य बनाने की एक निरंतर प्रतिबद्धता है, जिन्हें संरचनात्मक उत्पीड़न के परिणामस्वरूप p5.js समुदाय से बाहर रखा गया है। यह प्रतिबद्धता p5.js द्वारा प्रदान किए जाने वाले टूल और प्लेटफ़ॉर्म तक फैली हुई है। इसमें p5.js नेतृत्व की संरचना, निर्णय लेना और कार्य भी शामिल हैं। हम गति, विकास और प्रतिस्पर्धा की तकनीकी संस्कृति का विरोध करते हैं। हम सामूहिक देखभाल के कार्यों के रूप में जानबूझकर, धीमेपन, समायोजन और जवाबदेही को प्राथमिकता देते हैं।
 
-## पहुँच के प्रकार
+यहां पहुंच का अर्थ है p5.js को इनके लिए न्यायसंगत बनाना:
 
-बढ़ती पहुंच p5.js समुदाय के लोगों की संख्या के विस्तार पर केंद्रित नहीं है। यह p5.js को उपलब्ध कराने और उन लोगों के लिए स्वीकार्य करने पर ध्यान केंद्रित करता है, जिन्हें p5.js समुदाय (जानबूझकर या नहीं) और समान उपकरण और समुदायों से बाहर रखा गया है।
+- जो लोग अंग्रेजी के अलावा अन्य भाषाएँ बोलते हैं
+- काले, स्वदेशी, रंगीन लोग, और हाशिए पर रहने वाली जातीयता के लोग
+- समलैंगिक, समलैंगिक, उभयलिंगी, समलैंगिक, प्रश्न पूछने वाले, पैनसेक्सुअल और अलैंगिक लोग
```
Comment (Divyansh013) [inline contributor_docs/hi/access.md:15]: repetition doesn't sound good?

File: contributor_docs/hi/access.md [hunk 3]
```diff
@@ -1,40 +1,56 @@
...
 
-**कृपया इसे एक प्रारंभिक बिंदु माने** हम इस बारे में अधिक बातचीत को आमंत्रित करना चाहते हैं कि पहुँच का क्या अर्थ है और हम इसे कैसे प्राथमिकता दे सकते हैं।
+पहुंच बढ़ाना p5.js समुदाय में लोगों की संख्या का विस्तार करने पर केंद्रित नहीं है। यह उन लोगों के लिए p5.js को उपलब्ध कराने और उन तक पहुंच योग्य बनाने की एक निरंतर प्रतिबद्धता है, जिन्हें संरचनात्मक उत्पीड़न के परिणामस्वरूप p5.js समुदाय से बाहर रखा गया है। यह प्रतिबद्धता p5.js द्वारा प्रदान किए जाने वाले टूल और प्लेटफ़ॉर्म तक फैली हुई है। इसमें p5.js नेतृत्व की संरचना, निर्णय लेना और कार्य भी शामिल हैं। हम गति, विकास और प्रतिस्पर्धा की तकनीकी संस्कृति का विरोध करते हैं। हम सामूहिक देखभाल के कार्यों के रूप में जानबूझकर, धीमेपन, समायोजन और जवाबदेही को प्राथमिकता देते हैं।
 
-## पहुँच के प्रकार
+यहां पहुंच का अर्थ है p5.js को इनके लिए न्यायसंगत बनाना:
 
-बढ़ती पहुंच p5.js समुदाय के लोगों की संख्या के विस्तार पर केंद्रित नहीं है। यह p5.js को उपलब्ध कराने और उन लोगों के लिए स्वीकार्य करने पर ध्यान केंद्रित करता है, जिन्हें p5.js समुदाय (जानबूझकर या नहीं) और समान उपकरण और समुदायों से बाहर रखा गया है।
+- जो लोग अंग्रेजी के अलावा अन्य भाषाएँ बोलते हैं
+- काले, स्वदेशी, रंगीन लोग, और हाशिए पर रहने वाली जातीयता के लोग
```
Comment (Divyansh013) [inline contributor_docs/hi/access.md:14]: previous line was better for this

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

File: contributor_docs/hi/access.md (hunk 1 above)
Comment (jaiakash) [reply contributor_docs/hi/access.md:3]: Done

**Comment by jaiakash** [comment]
@Divyansh013 PTAL at the updated translation.

**Comment by Divyansh013** [comment]
In access.md file, can you please revert back the changes in line 13? The previous translation by you was better

**Comment by jaiakash** [comment]
> In access.md file, can you please revert back the changes in line 13? The previous translation by you was better

Done

**Review by Divyansh013 — APPROVED** [review]

**Comment by Divyansh013** [comment]
@TanviKumar Please review

**Review by TanviKumar — APPROVED** [review]

**Comment by jaiakash** [comment]
Hi @Qianqianye, Can you please merge this PR.
Thanks

**Comment by Qianqianye** [comment]
Looks great, thanks!

**Pull request merged**